packages/go_router_builder/example/lib/main.g.dart

```diff
@@ -52,7 +58,22 @@ extension $HomeRouteExtension on HomeRoute {
 }
 
 extension $FamilyRouteExtension on FamilyRoute {
-  static FamilyRoute _fromState(GoRouterState state) => FamilyRoute(
+  static FamilyRoute _fromState(GoRouterState state) => const FamilyRoute();
+
+  String get location => GoRouteData.$location(
+        '/',
+      );
+
+  void go(BuildContext context) => context.go(location, extra: this);
```

Comment:
ShellRoute should not have go/push/pushRelacement

Reply:
Hmm... It probably shouldn't have a `location` either.